webapp/.env.dist

```diff
@@ -1,6 +1,12 @@
 # The Application secret
 APP_SECRET=
 
+# Postgres user
+POSTGRES_USER=
+
+# Postgres Password
+POSTGRES_PASSWORD=
+
 # The database URL
 DATABASE_URL="postgresql://<user>:<password>@postgres:5432/app?serverVersion=15&charset=utf8"
```

Comment:
Don't you need to update (or remove) this line accordingly?

Reply:
This line is requried by Panther when running end to end tests